po/it.po

```diff
@@ -189,7 +189,7 @@ msgstr "Touchpad.\n"
 "Trascina su questa area per spostare il cursore del mouse.\n"
 "Premi a lungo per trascinare il cursore del mouse.\n\n"
 "Un semplice clic verrà inviato al dispositivo accoppiato.\n"
-"Sinistra, centrale, tasto destro e rotelle della ruota."
+"Pulsante sinistro, centrale, destro e rotellina di scorrimento."
```

Comment:
Ugh, that _source_ message is awful English to begin with, we should really rewrite that into... like... full sentences. But, Google Translate agrees your translation is more in line with the source text, referencing 'buttons' rather than 'clicks'. (And who am I to argue with Google Translate?)